packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec

```diff
@@ -75,6 +94,12 @@ EOF
 %{gem_instdir}/settings.d
 %{foreman_proxy_bundlerd_dir}/smart_proxy_ansible.rb
 %config %{foreman_proxy_settingsd_dir}/ansible.yml
+%attr(-,root,root) %{foreman_proxy_dir}/.ansible
+%attr(-,root,root) %{foreman_proxy_dir}/.ansible_galaxy
+%attr(-,root,root) %{foreman_proxy_dir}/.ansible.cfg
+%attr(-,foreman-proxy,foreman-proxy) %{_localstatedir}/lib/foreman_proxy/ansible
+%attr(-,foreman-proxy,foreman-proxy) %{_localstatedir}/lib/foreman_proxy/ansible_galaxy
+%attr(-,root,foreman-proxy) %{_sysconfdir}/foreman-proxy/ansible.cfg
```

Comment:
This should be marked as a config file so it's not overwritten on upgrades. Maybe also good to set the attr to `0640`?